### PR TITLE
Update eslint and dependencies

### PR DESCRIPTION
**Pull request by loay**

### CODE REVIEW

**Comment by loay** [comment]
@superkhau PTAL. Thanks.

**Comment by loay** [comment]
@slnode test please

File: package.json [hunk 1]
```diff
@@ -7,23 +7,30 @@
...
+    "should": "^10.0.0",
+    "sinon": "^1.17.5"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/strongloop/loopback-connector-mysql.git"
   },
+  "scripts": {
+  "lint": "eslint .",
+  "test": "./node_modules/.bin/mocha -R spec --timeout 10000 test/*.js",
```
Comment (superkhau) [inline package.json]: You don't need `./node_modules/.bin`. Also remove `-R spec`, keep it simple unless we need it later on.

```
"test": "mocha --timeout 10000 test/*.js"

**Comment by superkhau** [comment]
A few minor fixes, LGTM. No more review is needed once you fix those.

File: package.json [hunk 2]
```diff
@@ -8,23 +8,29 @@
...
     "rc": "^1.0.0",
-    "should": "^8.0.2",
+    "should": "^8.4.0",
     "sinon": "^1.15.4"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/strongloop/loopback-connector-mysql.git"
   },
+    "scripts": {
```
Comment (rmg) [inline package.json]: This should be merged with the existing `scripts`. As is, it is overwriting the previous one that includes a `pretest` that seeds the test db.

Comment (superkhau) [inline package.json]: I missed this one, @loay Please leave in the pretest.js script, why are you removing it?

**Comment by superkhau** [comment]
Fixes are good, LGTM. Please wait for CI and check for errors before merging.

**Comment by superkhau** [comment]
Squash too please.

File: package.json [hunk 3]
```diff
@@ -5,21 +5,25 @@
...
-    "loopback-datasource-juggler": "^2.28.0",
-    "mocha": "^2.1.0",
+    "bluebird": "^2.9.34",
+    "eslint": "^2.13.1",
+    "eslint-config-loopback": "^4.0.0",
+    "loopback-datasource-juggler": "^2.48.0",
+    "mocha": "^2.5.3",
     "rc": "^1.0.0",
-    "should": "^8.0.2",
+    "should": "^8.4.0",
```
Comment (superkhau) [inline package.json]: Do not upgrade deps as part of this PR. Remove please.

**Comment by superkhau** [comment]
I take back the LGTM, took a more detailed look, please do not upgrade dependencies as part of this PR.

**Comment by loay** [comment]
@superkhau Done PTAL

**Comment by superkhau** [comment]
Now it's LGTM. :ship:

**Comment by superkhau** [comment]
Remember to make sure any CI failures are unrelated to this PR.

**Comment by superkhau** [comment]
@jtary Can you help fix the strong-arc failures? It's causing our PRs to fail CI.

**Pull request merged**